frontend/src/routes/Credentials/CredentialsForm.tsx

```diff
@@ -250,6 +252,18 @@ export function CredentialsForm(props: {
     // Red Hat Cloud
     const [ocmAPIToken, setOcmAPIToken] = useState(providerConnection?.stringData?.ocmAPIToken ?? '')
 
+    function removeTrailingChar(url: string, char: string): string {
+        return _.trimEnd(url, char)
+    }
+
+    function removeLeadingTrailingSpaces(url: string): string {
+        return _.trim(url)
+    }
+
+    function cleanAnsibleUrl(url: string): string {
+        return removeTrailingChar(removeLeadingTrailingSpaces(url), '/')
+    }
+
```

Comment:
I guess the first 2 functions are just aliases... could just collapse them into `cleanAnsibleUrl`